site/gatsby-site/cypress/e2e/unit/risksApi.cy.js

```diff
@@ -0,0 +1,59 @@
+const { gql } = require('@apollo/client');
```

Comment:
I was thinking tests like these ones: https://github.com/responsible-ai-collaborative/aiid/blob/master/site/gatsby-site/cypress/e2e/unit/functions/promoteSubmissionToReport.cy.js

But this one is fine, we can add more later